rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/InterpretedLanguageContributions.java

```diff
@@ -370,8 +403,26 @@ public CompletableFuture<SummaryConfig> getOndemandSummaryConfig() {
     public InterruptibleFuture<@Nullable IValue> executeCommand(String command) {
         logger.debug("executeCommand({}...) (full command value in TRACE level)", () -> command.substring(0, Math.min(10, command.length())));
         logger.trace("Full command: {}", command);
-        return InterruptibleFuture.flatten(parseCommand(command).thenCombine(commandExecutor,
-            (cons, func) -> EvaluatorUtil.<@Nullable IValue>runEvaluator("executeCommand", eval, ev -> func.call(cons), null, exec, true, client)
+
+        return InterruptibleFuture.flatten(parseCommand(command).thenCombine(
+            commandExecutor,
+            (cons, func) -> {
+                
+                if (func == null) {
+                    logger.info("Command is being executed without a registered command executor; for language " + name);
```

Comment:
nit: in log4j, it's preffered to use `{}`templates instead of string concat. 

apart from that, I think this would warrant an `error` instead of `info`.

Reply:
Nothing crashes but it just does not have an implementation yet. That would be normal during DSL development, so I'm going for a warning.